pom.xml

```diff
@@ -74,7 +74,7 @@
         <jackson.version>2.12.5</jackson.version>
 
         <log4j.version>2.14.1</log4j.version>
-        <lucene.version>7.7.3</lucene.version>
+        <lucene.version>8.9.0</lucene.version>
```

Comment:
Add a comment above that the dependency version must be synchronised with Elasticsearch

Reply:
Good one, will do.